app/components/file-menu.js

```diff
@@ -33,13 +33,15 @@ export default Ember.Component.extend({
       prompt('Ctrl + C ;-)', window.location.href);
     },
     embed() {
-      let src = window.location.href.split("?")[0];
-      src += "?fullScreen=true";
-      let iframe = document.createElement("iframe");
+      let src = window.location.href.split('?')[0];
+      src += '?fullScreen=true';
+      let responsive = document.createElement('div');
+      responsive.style.cssText = 'position:relative;height:0;overflow:hidden;max-width:100%;padding-bottom:56.25%;'; // 16:9
```

Comment:
By setting the width of the containing element, ie. what they're embedding it into. With this change, the iframe takes up the full width of the "column" it's in and the height adjusts proportionally. [See here](https://css-tricks.com/NetMag/FluidWidthVideo/Article-FluidWidthVideo.php) for more info on how this works.

As it stands now, users cannot set the dimensions of the iframe unless they edit the markup before pasting into their own document.